.env.example

```diff
@@ -11,4 +11,7 @@ RECEIVE_HASHTAGS=csv of hashtags to watch for in tweets. Hash symbol is not need
 SIGNAL_MESSAGE_HEADERS=csv of Signal message headers to monitor. Example RESPONSE,DISPATCH,GENERIC MESSAGE
 AUTOSCAN_STATE_FILE_PATH=Unix path to store the statefile relative to the /app directory. Default is signal_scanner_bot/.autoscanner-state-file. This is an advanced parameter and likely should never be changed unless you have a specific need. If you do set this parameter it is important to store it in a place that will persist over container reloads, ie. the volumes mounted by the docker-compose file (currently signal-cli:/app/data/signal-cli, or ./signal_scanner_bot:/app/signal_scanner_bot).
 TESTING=True or False
-DEBUG=True or False
\ No newline at end of file
+DEBUG=True or False
+COMRADELY_CONTACT=signal group ID to send the message to
+COMRADELY_MESSAGE=contents of the message to send
+COMRADELY_TIME=time to send the message, needs to be in ISO format (e.g. 20:00:00)
```

Comment:
I think you mean UTC (specifically 24 hour clock UTC) here instead of ISO. At least, I think it will be easier for other people to understand that what we mean if we call it UTC.

Reply:
I definitely do mean ISO format here (since we're using the `fromisoformat` method, but now that I think about it this is UTC regardless so I should make not of it here in the example file (I thought it was machine TZ dependent but it isn't lol).